case-lib/lib.sh

```diff
@@ -672,6 +672,31 @@ is_zephyr()
     test "$znum" -gt 10
 }
 
+is_firmware_file_zephyr()
+{
+    local firmware_name firmware_path znum platform
+    # get the FW name and path
+    # TODO: optimize this part after driver can expose the FW path to userspace via debugfs
+    if is_ipc4; then
```

Comment:
```bash
    if [ is_ipc4 ]; then
```

Reply:
Sorry, is this really matter to you? We used it in this way in many places and there is no shellcheck warning.

Reply:
I just tried, if I do it like you suggested, the shellcheck will be failed.
```
In case-lib/lib.sh line 686:
    if [ is_ipc4 ]; then
         ^-----^ SC2078 (error): This expression is constant. Did you forget a $ somewhere?

```

Reply:
Sorry, this is fine, is_ipc4 is a function.